charts/galoy/values.yaml

```diff
@@ -87,7 +87,7 @@ redis:
     persistence:
       enabled: true
   auth:
-    enabled: false
+    password: password
```

Comment:
password is automatically generated?

Reply:
Nope. The redis chart differs from the mondo chart in this aspect. We need
to provide the password explicitly via the values.

Reply:
how is the password defined in that case currently? You'll manually set it?

Reply:
Yes, via the values.yaml. For testnet and mainnet we have our regular flow of injecting from the `configs` repo and for regtest we have it in the default values.yaml file